packages/cli/test/utils/runUtils.ts

```diff
@@ -17,20 +17,13 @@ export function findApiToken(dirpath: string): string {
 }
 
 export function expectDeepEquals<T>(a: T, b: T, message: string): void {
-  try {
-    expect(a).toEqual(b);
-  } catch (e) {
-    expect.fail(message);
-  }
+  expect(a).toEqualWithMessage(b, message);
 }
 
 /**
  * Similar to `expectDeepEquals` but only checks presence of all elements in array, irrespective of their order.
  */
 export function expectDeepEqualsUnordered<T>(a: T[], b: T[], message: string): void {
-  try {
-    expect(a.sort()).toEqual(b.sort());
```

Comment:
Did you find out what was the issue with this?

Reply:
You can't sort objects like that, see PR description.

See previous PR on why we need unordered equality checks 
- https://github.com/ChainSafe/lodestar/pull/5641

Reply:
Oh sure right, somehow I assumed we were iterating object keys here.